packages/headless/src/api/analytics/analytics.ts

```diff
@@ -194,6 +194,17 @@ export const getVisitorID = () =>
 
 export const historyStore = new history.HistoryStore();
 
+export const getPageID = () => {
+  const actions = historyStore.getHistory();
+  const lastPageView = actions.reverse().find((action) => {
+    return action.name === 'PageView';
+  });
```

Comment:
Since this function will be called fairly often and I'm not sure how many pageviews some clients may have logged, I think it may be worth using `lastIndexOf` (or something similar) and skipping `reverse()` for performance.

Reply:
Max page views is 10.

Reply:
+1 for lastIndexOf

Reply:
Oh thought it took a method as a second arg, bad memory.

Good as is then